.github/workflows/e2e-test-cluster.yml

```diff
@@ -0,0 +1,41 @@
+name: e2e Test Cluster
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+
+    - name: Check out code
+      uses: actions/checkout@v1
+
+    - name: Get dependencies
+      run: |
+        sudo apt-get install -y mysql-server mysql-client make unzip g++ etcd curl git wget
```

Comment:
@morgo how do we know which version of mysql is being used here?

Reply:
It is the one included in `ubuntu-latest`, so likely 8.0. I can change it to MySQL 5.7 from Oracle repos. For the local-example it does the same, but a matrix build probably makes more sense so we catch both.

Reply:
I looked into this further, it is 5.7:
```
mysql-client is already the newest version (5.7.27-0ubuntu0.18.04.1).
mysql-server is already the newest version (5.7.27-0ubuntu0.18.04.1).
```
The `ubuntu-latest` is 18.04. Assuming GitHub plans to bump this with new lts releases, this will change to 20.04, and MySQL 8.0 in April 2020. The fix then is to switch to the MySQL team repos, or consider upgrading our testsuite. Since MySQL 8.0 initializes slower, we will most likely chose the repo switch:
```
        wget https://dev.mysql.com/get/mysql-apt-config_0.8.13-1_all.deb
        sudo apt install -y ./mysql-apt-config_0.8*
        sudo sed -i 's/mysql-8.0/mysql-5.7/' /etc/apt/sources.list.d/mysql.list
        sudo apt-get update
        sudo apt-get install mysql-community-server mysql-community-client
```